src/main/java/com/example/SpringBootDemoProject/controller/DemoController.java

```diff
@@ -0,0 +1,15 @@
+package com.example.SpringBootDemoProject.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/demo")
+public class DemoController {
+
+    @GetMapping("/test") // URL for this endpoint will be : http://localhost:8080/demo/test
```

Comment:
don't add comments in file

Reply:
Hi I have added this comment for just infor purpose I'll remove it in my next pr